cardano_cli_helper.py

```diff
@@ -181,15 +185,13 @@ def getDraftTXSimple(txInList, returnAddr, recipientAddr, ttlSlot):
     return
 
 
-def getRawTxSimple(txInList, returnAddr, recipientAddr, lovelace_amount, lovelace_return, ttlSlot, fee):
+def getRawTxSimple(txInList, returnAddr, recipientAddr, lovelace_amount, ttlSlot, network, era='babbage-era'):
     print('Creating simple tx.raw...')
-    command = f'cardano-cli transaction build-raw \
-                --fee {fee} '
+    command = f'cardano-cli transaction build --{era} --{network} '
     for txIn in txInList:
         command += f'--tx-in {txIn} '
     command += f'--tx-out {recipientAddr}+{lovelace_amount} '
-    if lovelace_return > 0:
-        command += f'--tx-out {returnAddr}+{lovelace_return} '
+    command += f'--change-address {returnAddr} '
```

Comment:
this function was a bit simplified by using the more user friendly `transaction build`

Reply:
I still need to maintain legacy, because tx build sends the TxIx of the receiver first, and the fuel faucet is set to expect the sender first.

Reply:
Actually i think we're good from this side, the fuel faucet has an issue. It should be looking for TxIx 1 not 0 for the sender. Not sure how that was changed.